### PR TITLE
feat: Add config env to use tag or digest in image ref passed to scannercli

### PR DESCRIPTION
**Pull request by danielpacak**

### CODE REVIEW

File: pkg/aqua/command.go [hunk 1]
```diff
@@ -46,6 +61,11 @@ func (c *command) Exec(imageRef string) (report ScanReport, err error) {
 		}
 	}()
 
+	image := imageRef.WithDigest()
+	if c.cfg.UseImageTag {
+		image = imageRef.WithTag()
```
Comment (knqyf263) [inline pkg/aqua/command.go]: Isn't it necessary to test `WithTag`? In tests, `UseImageTag` always appears to be false.

File: pkg/scanner/adapter.go [hunk 2]
```diff
@@ -23,8 +22,11 @@ func NewAdapter(command aqua.Command, transformer Transformer) Adapter {
 }
 
 func (s *adapter) Scan(req harbor.ScanRequest) (harbor.ScanReport, error) {
-	imageRef := fmt.Sprintf("%s:%s", req.Artifact.Repository, req.Artifact.Tag)
-	aquaScanReport, err := s.command.Exec(imageRef)
+	aquaScanReport, err := s.command.Exec(aqua.ImageRef{
+		Repository: req.Artifact.Repository,
+		Tag:        req.Artifact.Tag,
+		Digest:     req.Artifact.Digest,
```
Comment (knqyf263) [inline pkg/scanner/adapter.go]: Is this digest always passed?

Comment (knqyf263) [reply pkg/scanner/adapter.go]: Because when it isn't passed, an image becomes like `alpine@`

Comment (danielpacak) [reply pkg/scanner/adapter.go]: Good question. Both Tag and Digest are passed from Harbor to the Adatper. Those are mandatory properties validated by HTTP handler. If they're blank the HTTP handler will reply with 422 Unprocessable Entity status code.

Comment (knqyf263) [reply pkg/scanner/adapter.go]: I see.

File: pkg/aqua/command.go (hunk 1 above)
Comment (knqyf263) [reply pkg/aqua/command.go]: This will be done by a dedicated PR.

**Pull request merged**